Tasks/ANT/anttask.ts

```diff
@@ -78,14 +80,28 @@ function processAntOutputLine(line) {
 
 async function doWork() {
 
-    function enableCodeCoverage() {
+    function execEnableCodeCoverage(): Q.Promise<string> {
+        return enableCodeCoverage()
+            .then(function (resp) {
+                tl.debug("Enabled code coverage successfully");
+                return "CodeCoverage_9064e1d0";
+            }).catch(function (err) {
+                tl.warning("Failed to enable code coverage: " + err);
+                return "";
+            });
+    };
+
+    function enableCodeCoverage(): Q.Promise<any> {
```

Comment:
the reporting task which we add varies for different tools and slightly different. It's the one reason, I kept enabling part to common module and calling code to respective build tools